exporter/collector/config.go

```diff
@@ -46,8 +46,8 @@ type Config struct {
 	ProjectID               string            `mapstructure:"project"`
 	UserAgent               string            `mapstructure:"user_agent"`
 	ImpersonateConfig       ImpersonateConfig `mapstructure:"impersonate"`
-	LogConfig               LogConfig         `mapstructure:"log"`
 	TraceConfig             TraceConfig       `mapstructure:"trace"`
+	LogConfig               LogConfig         `mapstructure:"log"`
```

Comment:
any reason why this was moved?  Technically, moving within a struct is a breaking change in go.

Reply:
fieldalignment wanted this moved when I added the new fields. Guess it didn't care once I removed them. moved it back